config/default.yml

```diff
@@ -110,6 +110,14 @@ RSpec:
       - subject
       - subject!
 
+Metrics/BlockLength:
+  inherit_mode:
```

Comment:
`rubocop-rails` goes as far as [setting `Exclude` to `merge` globally](https://github.com/rubocop/rubocop-rails/blob/944394b9a45c82137cd9af8852ef43b3cd472265/config/default.yml#L3).